src/commands/git-metadata/library.ts

```diff
@@ -17,6 +17,21 @@ export const isGitRepo = async (): Promise<boolean> => {
   }
 }
 
+// getGitCommitInfo returns the current [repositoryURL, commitHash]. If parameter
+// filterAndFormatGitRepoUrl == true, the repositoryURL will have sensitive information filtered and
+// git prefix normalized.
+// ("git@github.com:" and "https://github.com/" prefixes will be normalized into "github.com/")
+export const getGitCommitInfo = async (filterAndFormatGitRepoUrl?: boolean): Promise<[string, string]> => {
```

Comment:
will be used in datadog-cdk and the serverless plugin to get the commit sha and remote repo url